catalog/glue/glue.go

```diff
@@ -747,50 +743,61 @@ func filterTableListByType(database string, tableList []types.Table, tableType s
 	return filtered
 }
 
-func buildGlueTableInput(ctx context.Context, database string, tableName string, staged *table.StagedTable, cat *Catalog) (*types.TableInput, error) {
-	glueTable, err := cat.getTable(ctx, database, tableName)
-	if err != nil {
-		return nil, err
-	}
-
-	glueProperties := prepareProperties(staged.Properties(), staged.MetadataLocation())
-	description := staged.Properties()["comment"]
-	if description == "" {
-		description = aws.ToString(glueTable.Description)
-	}
-	existingColumnMap := map[string]string{}
-	for _, column := range glueTable.StorageDescriptor.Columns {
-		existingColumnMap[*column.Name] = *column.Comment
-	}
-	var glueColumns []types.Column
-	for _, column := range schemaToGlueColumns(staged.Metadata().CurrentSchema(), true) {
-		col := types.Column{
-			Name:    column.Name,
-			Comment: column.Comment,
-			Type:    column.Type,
+func constructParameters(staged *table.Table, previousGlueTable *types.Table) map[string]string {
+	var parameters map[string]string
+	if previousGlueTable != nil {
+		parameters = previousGlueTable.Parameters
```

Comment:
is it possible for this to be null? should we use `maps.Copy` instead so that we don't modify the `previousGlueTable`?